tests/common/devices.py

```diff
@@ -268,6 +268,21 @@ def _get_platform_info(self):
                 result["hwsku"] = line.split(":")[1].strip()
             elif line.startswith("ASIC:"):
                 result["asic_type"] = line.split(":")[1].strip()
+
+        if result["platform"]:
+            platform_file_path = os.path.join('/usr/share/sonic/device', result["platform"], 'platform.json')
+
+            try:
+                out = self.command("cat {}".format(platform_file_path))
+                platform_info = json.loads(out["stdout"])
+                result["chassis"] = platform_info["chassis"]
```

Comment:
Rather than explicitly adding `"chassis"` to `result` here, can you add all the elements parsed from the file? This way the code will not need to be modified if/when more elements are added to the file.

Reply:
fixed